python/cudf/cudf/tests/test_indexing.py

```diff
@@ -1898,24 +1897,25 @@ def test_iloc_column_boolean_mask_issue_13265():
     assert_eq(expect, actual)
 
 
-@pytest.mark.xfail(
-    reason="https://github.com/rapidsai/cudf/issues/13266 "
-    "and https://github.com/rapidsai/cudf/issues/13273"
-)
 def test_iloc_repeated_column_label_issue_13266():
     # https://github.com/rapidsai/cudf/issues/13266
     # https://github.com/rapidsai/cudf/issues/13273
     df = pd.DataFrame(np.arange(4).reshape(2, 2))
     cdf = cudf.from_pandas(df)
 
     expect = df.iloc[:, [0, 1, 0]]
-    actual = cdf.iloc[:, [0, 1, 0]]
-    assert_eq(expect, actual)
+    with pytest.raises(NotImplementedError):
+        actual = cdf.iloc[:, [0, 1, 0]]
+        assert_eq(expect, actual)
```

Comment:
Does the `assert_eq` raise, or does the `iloc` raise? The `pytest.raises` should be narrowly scoped to the line that raises. If we're solely testing for an error in cudf (due to repeated columns being unsupported), it doesn't make a lot of sense to compute the `expect` value and try to assert things about it.

Reply:
it's the `iloc`. I will remove the other line.